templates/layouts/configs/_footer.html

```diff
@@ -6,20 +6,42 @@
 } %}
 
 {% if use_transactional_footer %}
-  {% do footer | setAttribute("rows", [
+  {% do footer | setAttribute("cols", [
+      {
+          "itemsList": [
+              {
+                    "text": _('Contact us'),
+                    "url": '/contact-us',
+                    "target": "_blank"
+              },
+              {
+                    "text": _('Help'),
+                    "url": 'http://census.gov.uk/help/how-to-answer-questions/online-questions-help-ew/',
```

Comment:
Should use `https`. 
This page doesn't exist yet?

Reply:
No but it will do. Ill remove for now